include/erlcloud_ecs.hrl

```diff
@@ -38,14 +38,14 @@
 }).
 
 -record(ecs_deployment, {
-    created_at :: undefined | pos_integer(),
+    created_at :: undefined | pos_integer() | float(),
```

Comment:
This seems wrong, and I guess I did it to have tests not complain.

Reply:
I'm going to fix the tests for this. It'll revert an API change that I did, but it's not a breaking change, since the expectation was already previously a `pos_integer()`.

Reply:
Done in ba02c27